README.rst

```diff
@@ -359,6 +359,55 @@ all contained in one binary.
 So if feasible, aim at static linking, which is currently only possible with
 AnaConda Python on non-Windows.
 
+Windows Standalone executables and dependecies
+----------------------------------------------
+
+The process of making Standalone executables for Windows traditionnaly involves
+using an external dependency walker in order to copy necessary libraries along
+with the compiled executables to the distribution folder.
+Usng the external dependency walker is quite time consuming, and may copy
+some unnecessary libraries along the way (better have too much than missing).
+Since Nuitka 0.6.2, there's an experimental alternative internal dependency
+walker that relies on pefile which analyses PE imports of executables / libraries.
+This implementation shall create smaller Standalone distributions since it won't
+include Windows' equivalent of the standard library, and will speed-up first
+Nuitka compilations by an order of magnitude.
+In order to use it, make sure you have pefile installed via ```python -m pip install pefile```
+Once installed, you may enable the internal dependency walker by using the
+following switch:
+
+.. code-block:: bash
+
+    python -m nuitka --standalone --experimental=use_pefile myprogram.py
+
+
+.. note::
+
+    The pefile dependency walker will test all dependencies of the distribution folder.
+
+    Optionnaly, it is also possible to check all recursive dependencies of included libraries
+    using the following switch along with the above one:
+
+.. code-block:: bash
+
+    python -m nuitka --standalone --experimental=use_pefile --experimental=use_pefile_recurse myprogram.py
+
+
+.. note::
+
+    Some modules may have hidden dependencies outside of their directory. In order for
+    the pefile dependency walker to find them, you may also scan the whole site-packages
+    directory for missing dependencies using the following switch along with the two above:
+
+.. code-block:: bash
+
+    python -m nuitka --standalone --experimental=use_pefile --experimental=use_pefile_recurse --experimental=use_pefile_fullrecurse myprogram.py
```

Comment:
I have to see how this works. For `depend.exe` it seemed barely necessary. I think it uses all sub-directories of the package we are working for.

Also more search directories, why should they make things slower in a way we notice. I will however check that out.

Reply:
It is useful to scan all directories for missing dll files.
I had a case with pywin32, where pythoncom37.dll is in pywin_system32 and not in the win32 module folder. This resolves such issues.
Of course for pywin32, since most windows executables will use it, I made a hard coded fix.

And yes, it's very slow since it will check the whole site-packages root for every dependency :)